src/interaction/scroll.ts

```diff
@@ -20,9 +19,12 @@ function recompute(event: UIEvent = null): void {
     let x = Math.round((eventTarget as HTMLElement).scrollLeft);
     let y = Math.round((eventTarget as HTMLElement).scrollTop);
     let id = getId(eventTarget as Node);
-    target.observe(id);
     let current: ScrollData = {target: id, x, y, time: time()};
 
+    // We don't send any scroll events if the current position is at the top (0, 0)
```

Comment:
>current [](start = 46, length = 7)

Reword to "We don't send any scroll events if this is the first event and the current position is top (0,0).

Reply:
Done